client/components/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import SummaryOverall from './summaryOverall';
+import GraphOverall from './graphOverall';
+import SummaryRecent from './summaryRecent';
+import GraphRecent from './graphRecent';
+
+const slashes = window.location.href.split('/');
+const gameid = slashes[slashes.length - 1].split('?')[0].split('#')[0];
```

Comment:
I saw your comment in the Slack channel - what's the reason for this URL change? I think the only front-facing aspect of your service would be the frontend itself, no? Since the proxy server would handle URL display

Reply:
I might not fully understand this, but my goal was to be able to have the React components pull the gameId from the topmost URL, so in order to test this, I wanted to format my module's URL the same way. 

http://hosted.url.com/game/76

I didn't think this was a change - my impression was that this was the format of the topmost URL. If I'm wrong and it was going to be something else, I'd happily adjust.